iFixit/react-commerce
# Refactor header components

fixes #245 

## Changelog

- Export user model as an `auth-sdk` package (this could be later expanded with other auth related stuff)
- Export header components from `ui` package
- Create a `helpers` package to share reusable helpers
- Refactored layout and header components to use components from `ui` package

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com)
2. Visit all parts product list
3. Verify that everything works the same way as for the main branch
